src/imgkit/__init__.py

```diff
@@ -1,5 +1,8 @@
 # -*- coding: utf-8 -*-
-"""Wkhtmltopdf python wrapper to convert html to image using the webkit rendering engine and qt"""
+"""
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>D205 1 blank line required between summary line and description